packages/cozy-procedures/src/components/Amount.jsx

```diff
@@ -22,32 +25,35 @@ class Amount extends React.Component {
       'procedureData.amount.default'
     )
     return (
-      <div>
-        <Topbar title={t('amount.title')} />
-        <Title className="u-mb-2 u-ta-center">{t('amount.subtitle')}</Title>
+      <>
+        <PageLayout>
+          <PageContent>
+            <Topbar title={t('amount.title')} />
+            <Title className="u-mb-2 u-ta-center">{t('amount.subtitle')}</Title>
 
-        <div className="u-mb-1">
-          <Label htmlFor="amount-input">{t('amount.label')}</Label>
-          <Input
-            type="number"
-            id="amount-input"
-            min={min}
-            max={max}
-            placeholder={defaultValue.toString()}
-            value={amount === null ? '' : amount}
-            onChange={e => updateAmount(parseInt(e.target.value))}
-          />
-        </div>
-
-        <div>
-          <Button
-            label={t('confirm')}
-            extension="full"
-            onClick={router.goBack}
-            disabled={amount === null}
-          />
-        </div>
-      </div>
+            <div className="u-mb-1">
+              <Label htmlFor="amount-input">{t('amount.label')}</Label>
+              <Input
+                type="number"
+                id="amount-input"
+                min={min}
+                max={max}
+                placeholder={defaultValue.toString()}
+                value={amount === null ? '' : amount}
+                onChange={e => updateAmount(parseInt(e.target.value))}
+              />
+            </div>
+          </PageContent>
+          <PageFooter>
+            <Button
+              label={t('confirm')}
+              extension="full"
+              onClick={router.goBack}
+              disabled={amount === null}
+            />
+          </PageFooter>
+        </PageLayout>
+      </>
```

Comment:
The fragment is not needed since everything is already wrapped in the `PageLayout`

Reply:
Good catch, thanks !